cms/conf.py

```diff
@@ -174,41 +173,50 @@ def __init__(self):
         self.telegram_bot_token = None
         self.telegram_bot_chat_id = None
 
-        self.log_dir = os.path.join("/", "var", "local", "log", "cms")
-        self.cache_dir = os.path.join("/", "var", "local", "cache", "cms")
-        self.data_dir = os.path.join("/", "var", "local", "lib", "cms")
-        self.run_dir = os.path.join("/", "var", "local", "run", "cms")
-        paths = [
-            os.path.join("/", "usr", "local", "etc", "cms.toml"),
-            os.path.join("/", "etc", "cms.toml"),
-        ]
+        # Try to find CMS installation root from the venv in which we run
+        self.base_dir = sys.prefix
+        if self.base_dir == '/usr':
+            # This looks like a system-wide installation, so try backward-compatible paths
+            # FIXME: Do we want to support this?
```

Comment:
I have a mild preference for not supporting this, and making things simpler.

Reply:
OK. I removed support for running outside a venv. I simplified reading of configuration, made a missing configuration file always fatal and removed the "Using configuration file..." message upon startup.